tests/unit/test_core/test_interface.py

```diff
@@ -1,66 +1,69 @@
 """Interface tests."""
+# pylint: disable=attribute-defined-outside-init
 import logging
 import pickle
-import unittest
 from unittest.mock import MagicMock, Mock
 
+import pytest
 from pyof.v0x04.common.port import PortFeatures
 
 from kytos.core.common import EntityStatus
-from kytos.core.interface import TAG, UNI, Interface, TAGType
+from kytos.core.exceptions import (KytosSetTagRangeError,
+                                   KytosTagtypeNotSupported)
+from kytos.core.interface import TAG, UNI, Interface
 from kytos.core.switch import Switch
 
 logging.basicConfig(level=logging.CRITICAL)
 
 
-class TestTAG(unittest.TestCase):
+class TestTAG():
     """TAG tests."""
 
-    def setUp(self):
+    def setup_method(self):
         """Create TAG object."""
-        self.tag = TAG(1, 123)
+        self.tag = TAG('vlan', 123)
 
-    def test_from_dict(self):
+    async def test_from_dict(self):
```

Comment:
If you don't have any async awaitable it's not needed to turn this into a coroutine, so you could leave the `async` part, there's no harm though, but if doesn't need `async` then you can keep it as just as a `def`.

Reply:
Got it.